etc/helm/pachyderm/templates/tests/test-connection.yaml

```diff
@@ -20,6 +20,13 @@ spec:
       image: "alpine"
       command: ['/bin/sh', '-c', '--']
       args: ['while ! nc -vz pachd-peer 30653 ; do sleep 1 ; done']
+      resources:
+          limits:
+            cpu: 250m
+            memory: 500Mi
```

Comment:
I bet this could be 50Mi like the request.  Hopefully we never use that much memory.

Reply:
oops. typo